### PR TITLE
update wj-mcat developer information

### PR DESCRIPTION
**Pull request by wj-Mcat**
update myself-introduction.

### CODE REVIEW

**Comment by wj-Mcat** [comment]
@huan  Waiting for your review.

File: _developers/wj-Mcat.md [hunk 1]
```diff
@@ -2,16 +2,27 @@
...
+![love to see beautiful scenery](/assets/developers/wj-Mcat/scenery.jpeg)
 
 ## Contact me
 
-- Github: <https://github.com/wj-Mcat>
-- Email：<>
-- Website: <>
+- Github: [https://www.github.com/wj-Mcat](https://www.github.com/wj-Mcat)
+- Wechat: [pure-_--love](/imgs/weixin.jpeg)
+- WeiBo: [猫神战](https://weibo.com/u/3877178684)
```
Comment (huan) [inline _developers/wj-Mcat.md:24]: You can put your Weibo URL to the `twitter` in front matter.

**Review by huan — APPROVED** [review]
It's a great bio for your first version!

Please consider my suggestion in the comment, and feel free to help yourself to merge this PR when you feel comfortable with it!

File: _developers/wj-Mcat.md (hunk 1 above)
Comment (wj-Mcat) [reply _developers/wj-Mcat.md:24]: This will be updated later.

**Comment by wj-Mcat** [comment]
@huan It seems that merging must be after `Approve`. But I can't approve myself code.

**Review by huan — APPROVED** [review]
LGTM

**Comment by huan** [comment]
Oh, I thought I was approved but it seems not.

Now should be OK, please try again.

**Pull request merged**